src/ape/api/accounts.py

```diff
@@ -164,24 +210,54 @@ class AccountContainerAPI:
     @property
     @abstractmethod
     def aliases(self) -> Iterator[str]:
-        ...
+        """
+        List all available aliases.
+
+        Returns:
+            Iterator[str]: List of aliases.
+        """
 
     @abstractmethod
     def __len__(self) -> int:
-        ...
+        """
+        Number of aliases.
+
+        """
 
     @abstractmethod
     def __iter__(self) -> Iterator[AccountAPI]:
-        ...
+        """
+        List all accounts.
+
+        Returns:
+            Iterator[AccountAPI] (``List[str]``)
+        """
 
     def __getitem__(self, address: AddressType) -> AccountAPI:
+        """
+        Raw input will be used to iterate and match the address.
+
+        Returns:
+            AccountAPI (:class:`~ape.api.accounts.AccountAPI`)
+        """
         for account in self.__iter__():
             if account.address == address:
                 return account
 
         raise IndexError(f"No local account {address}.")
 
     def append(self, account: AccountAPI):
+        """
+        Add account the iterable list of accounts in Account Container.
+
+        Args:
+            account (:class:`~ape.api.accounts.AccountAPI`): Requires contract type.
+
+        Flags:
```

Comment:
What is `Flags` ?

Reply:
I was sure how to elaborate over why our code verify certain parts.